### ISSUE TITLE
about sceneflow pre-trained models

### ISSUE THREAD

**Issue by xubin1994** [feihuzhang/GANet#46]
Hi，I use your source code to train on the sceneflow dataset, batchsize = 4 on four V100 GPUs, but the results obtained are not as good as the ones you provided in the real image test. Will the impact of batchsize be so big? Or what additional optimizations have you made to the sceneflow model you provided?
the result test on my model(sceneflow-10epochs):
![image](https://user-images.githubusercontent.com/28856641/64255216-7beee600-cf53-11e9-88cc-3279037c2abd.png)

and the result test on you model(sceneflow-10epochs):
![image](https://user-images.githubusercontent.com/28856641/64255225-814c3080-cf53-11e9-9f60-45b1a623286e.png)

What puzzles me the most is that my model size is 77307KB, but the model size you provide is 77318KB. What causes the parameter size inconsistency?

**Comment by feihuzhang** [comment]
Batchsize is always one of the most important parameter for any models. 4 is too small, better to use 8-24. Another point is the crop_size. Larger is better, especially for the crop_width.

**Comment by xubin1994** [comment]
> Batchsize is always one of the most important parameter for any models. 4 is too small, better to use 8-24. Another point is the crop_size. Larger is better, especially for the crop_width.

My device memory is not enough to run too big batch_size. Is it theoretically possible to train the smaller batch_size more times to achieve the same accuracy? Is the method for determining the best batchsize for the network only to test different parameters for verification?

**Comment by xubin1994** [comment]
I use someone else's device to reimplement the sceneflow model, batchsize = 8, crop-shape = 240 * 624, and change the sync_bn to un-syncbn "torch.nn.BatchNorm(origin sync_bn caused some problem), but still can't get the same result as your model, my results are as follows:
![image](https://user-images.githubusercontent.com/28856641/64528728-cac5c280-d33b-11e9-8cca-1daa226805fa.png)
Do you use any optimizations for sync_bn?And this time  my model size is 77312KB,not 77307KB as before.

**Comment by feihuzhang** [comment]
It seems to be much better now.
For sync_bn, refer to #44 #35, if you met the similar problem.
sync_bn should perform better than normal bn.

If you could not solve the syn_bn problem, try to use a much larger crop_size.
e.g  288*672, 240*720 or any others.
You can also train more epoches, 12 or even 15 epoches to get better results.
